src/Umbraco.Web.UI.Client/src/common/services/tinymce.service.js

```diff
@@ -524,7 +524,7 @@ function tinyMceService($rootScope, $q, imageHelper, $locale, $http, $timeout, s
      */
     defaultPrevalues: function () {
       var cfg = {};
-      cfg.toolbar = ["ace", "styles", "bold", "italic", "alignleft", "aligncenter", "alignright", "bullist", "numlist", "outdent", "indent", "link", "umbmediapicker", "umbblockpicker", "umbembeddialog"];
+      cfg.toolbar = ["ace", "styles", "bold", "italic", "alignleft", "aligncenter", "alignright", "bullist", "numlist", "outdent", "indent", "link", "umbmediapicker", "umbmacro", "umbembeddialog"];
```

Comment:
@bergmania & @KennethJakobsen: Instead of replacing `umbmacro` with `umbblockpicker`, wouldn't it be nicer to have both of them enabled by default? Or maybe even better, to automatically enable it when available blocks are added to the configuration?

Reply:
We're moving towards phasing out macros entirely, so having macros default enabled for new RTEs seems counter productive to that end.

As for automatically adding the block picker toolbar option when blocks are configured for the RTE.... that might be a good idea. I'll bring it to the team. Or, if nothing else then the config icon should probably be updated to match that of the RTE toolbar:

![image](https://github.com/umbraco/Umbraco-CMS/assets/7405322/a2697060-c0e3-4c3c-b6b2-d2b4c8d04252)

![image](https://github.com/umbraco/Umbraco-CMS/assets/7405322/d6a641d2-ad80-4e2b-856f-44850b0ef509)